backend-config/types.go

```diff
@@ -121,7 +120,8 @@ func (c *ConfigT) DestinationsMap() map[string]*DestinationT {
 }
 
 type Settings struct {
-	DataRetention DataRetention `json:"dataRetention"`
+	DataRetention     DataRetention `json:"dataRetention"`
+	EventAuditEnabled bool          `json:"eventAuditEnabled"`
```

Comment:
Have we verified that data is unmarshalling into this struct automatically ?

Reply:
yes